.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       matrix:
         python-version: ['3.6.x', '3.7.x', '3.8.x']
         pg-version: ['11', '12']
-        django-version: ['2.1.15', '2.2.16']
+        django-version: ['2.2.24', '3.2.6']
```

Comment:
these are the two newest LTS versions.. so maybe that's what smartmin should aim to support from now on